src/components/look-controls.js

```diff
@@ -54,11 +57,21 @@ module.exports.Component = registerComponent('look-controls', {
     var data = this.data;
     if (!data.enabled) { return; }
     this.controls.standing = data.standing;
+    this.controls.userHeight = this.getUserHeight();
     this.controls.update();
     this.updateOrientation();
     this.updatePosition();
   },
 
+  /**
+   * Return user height to use for standing poses, where a device doesn't provide an offset.
+   */
+  getUserHeight: function () {
+    var headEl = this.data.headElement || this.el.sceneEl.camera.el;
```

Comment:
What are the cases where the headElement is not the active camera?

Reply:
(e.g. spectator camera?)